pkg/kubelet/network/dns/dns.go

```diff
@@ -116,7 +116,8 @@ func (c *Configurer) getClusterDNS(pod *v1.Pod) ([]net.IP, error) {
 	if len(c.serviceListers) > 1 {
 		klog.V(4).Infof("Need locate which serviceLister is correct one in multi-servceListers")
 		// To get correct index of serviceLister based on Pod's tenant information
-		index = c.clientManager.PickClient(pod.Tenant)
+		// todo: remove param of tenant after pod specific origin tracking impl
```

Comment:
Can this todo be done now? Or just leave tenant for accuracy?

Reply:
already did. will remove the obsolete comment. thanks for pointing it out.